app/src/main/java/ca/bc/gov/bchealth/ui/comment/CommentsViewModel.kt

```diff
@@ -57,6 +57,37 @@ class CommentsViewModel @Inject constructor(
         }
     }
 
+    fun addComment(parentEntryId: String, userProfileId: String?, comment: String) =
```

Comment:
userProfileId is hdId, which is available at repo level. Can we remove it from here?